streamer/src/nonblocking/quic.rs

```diff
@@ -1152,7 +1152,16 @@ pub mod test {
         for _ in 0..num_bytes {
             s1.write_all(&[0u8]).await.unwrap();
```

Comment:
Seems like now this `unwrap()` is failing:

```
thread 'quic::test::test_quic_server_multiple_writes' panicked at 'called `Result::unwrap()` on an `Err` value: Stopped(0)', streamer/src/nonblocking/quic.rs:1153:40
stack backtrace:
   0: rust_begin_unwind
             at /rustc/11ebe6512b4c77633c59f8dcdd421df3b79d1a9f/library/std/src/panicking.rs:575:5
   1: core::panicking::panic_fmt
             at /rustc/11ebe6512b4c77633c59f8dcdd421df3b79d1a9f/library/core/src/panicking.rs:65:14
   2: core::result::unwrap_failed
             at /rustc/11ebe6512b4c77633c59f8dcdd421df3b79d1a9f/library/core/src/result.rs:1791:5
   3: core::result::Result<T,E>::unwrap
             at /rustc/11ebe6512b4c77633c59f8dcdd421df3b79d1a9f/library/core/src/result.rs:1113:23
   4: solana_streamer::nonblocking::quic::test::check_multiple_writes::{{closure}}
             at ./src/nonblocking/quic.rs:1153:13
```

I wonder if there is something deeper that needs to be fixed here.
Like making sure that the receiver shutdown is property controlled, and does not race with the thread that is sending the data.

Reply:
Sorry for repetition.
GitHub didn't show the previous last 24 hours of discussions on this issue, it seems.
Only now I see that you have already figured it out.